tools/run_tests/xds_k8s_test_driver/README.md

```diff
@@ -269,7 +227,44 @@ envsubst < config/local-dev.cfg.example > config/local-dev.cfg
 
 Learn more about flagfiles in [abseil documentation](https://abseil.io/docs/python/guides/flags#a-note-about---flagfile).
 
-### Helper scripts
+## Test suites
+
+See the full list of available test suites in the [`tests/`](https://github.com/grpc/grpc/tree/master/tools/run_tests/xds_k8s_test_driver/tests) folder. 
+
+### xDS Baseline Tests
+
+Test suite meant to confirm that basic xDS features work as expected. Executing
+it before other test suites will help to identify whether test failure related
+to specific features under test, or caused by unrelated infrastructure
+disturbances.
+
+```shell
+# Help
+python -m tests.baseline_test --help
+python -m tests.baseline_test --helpfull
```

Comment:
s/python/python3/g

Reply:
This assumes virtualenv is active, so `python` will point to the correct version.